glib-macros/src/lib.rs

```diff
@@ -300,6 +301,34 @@ pub fn gboxed_derive(input: TokenStream) -> TokenStream {
     gen.into()
 }
 
+/// Derive macro for defining a [`SharedType`]`::get_type` function and
+/// the [`glib::Value`] traits.
+///
+/// # Example
+///
+/// ```
+/// use glib::prelude::*;
+/// use glib::subclass::prelude::*;
+///
+/// #[derive(Clone, Debug, PartialEq, Eq)]
+/// struct MySharedInner {
+///   foo: String,
+/// }
+/// #[derive(Clone, Debug, PartialEq, Eq, glib::Shared)]
+/// #[shared(type_name = "MyShared")]
+/// struct MyShared(std::sync::Arc::<MySharedInner>);
```

Comment:
```suggestion
/// struct MyShared(std::sync::Arc<MySharedInner>);
```

Reply:
Then there's only this one and we can get it merged once @bilelmoussaoui or @GuillaumeGomez reviewed it :)

Reply:
Ops! Sorry, I missed this one, should be fixed now